docs/api/session.md

```diff
@@ -916,6 +916,9 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
         Specifying a loopback device will capture system audio, and is
         currently only supported on Windows. If a WebFrameMain is specified,
         will capture audio from that frame.
+      * `enableLocalEcho` Boolean (optional) - If `audio` is a [WebFrameMain](web-frame-main.md)
+         and if this is set to `false`, then prevents input from being echoed in the local output
+         stream (e.g., speaker). Defaults to `false`.
```

Comment:
Would you mind rephrasing this? Perhaps describe what the default behavior is, and then describe what will happen by setting it to `true`.

Also, I'm not quite sure what "input being echoed in the local output stream" means. Do you mean audio input (i.e. from a microphone)? Could use some clarification!

Reply:
The closest documentation I could find that might be related to the flag is this:

https://bugs.chromium.org/p/chromium/issues/detail?id=647801

So I adapted the documentation to reflect that, and also provide a real-world usage example (which is my own use-case), which may help when this would be necessary.